src/components/Navigation/LanguageDropdown.js

```diff
@@ -9,17 +9,22 @@ import MenuItem from 'react-bootstrap/lib/MenuItem';
 import DropdownButton from 'react-bootstrap/lib/DropdownButton';
 import {getAvailableLanguages} from '../../utils/filterUtils';
 import styles from './LanguageDropdown.scss';
-import {getTranslateFilter} from '../../selectors/translate';
 import Flag from '../Flag';
 
 const availableLanguages = getAvailableLanguages();
 
-const LanguageItem = ({language, translateFilter, onlyFlag}) => (
+const languages = {
+  nb: 'Bokmål',
+  en: 'English',
+  nn: 'Nynorsk',
+};
+
+const LanguageItem = ({language, onlyFlag}) => (
   // Note that the block with "float" (the flag) must be first in the containing div
   <div className={styles.languageItemContainer}>
-    <Flag language={language}/>
+    <Flag {...{language}}/>
     <span className={styles.language + (onlyFlag ? ' ' + styles.onlyFlag : '')}>
-      {translateFilter('language', language)}
+      {languages[language]}
```

Comment:
Hva om vi skriver
```
{languages[language] || language}
```
her? Hvis det kommer et nytt språk, e.g. "is", så står det "is" i stedet for blankt?